src/main/scala/is/hail/annotations/UnsafeRow.scala

```diff
@@ -30,27 +66,92 @@ class UnsafeIndexedSeqAnnotation(
 
   override def write(kryo: Kryo, output: Output) {
     kryo.writeObject(output, ttBc)
-    val enc = new Encoder(new DataOutputStream(output))
+    val enc = new Encoder()
     enc.writeRegionValue(ttBc.value.typ, region, aoff)
+    kryo.writeObject(output, enc.outOff)
+    kryo.writeObject(output, enc.outMem) // FIXME
```

Comment:
????

Reply:
It's copying the full memory for each unsafeindexedSeq, right?